src/Polymake.jl

```diff
@@ -112,5 +115,8 @@ include("polynomial.jl")
 
 include("polymake_direct_calls.jl")
 
-include("generate_applications.jl")
+Base.CoreLogging.with_logger(Base.CoreLogging.NullLogger()) do
```

Comment:
I'd like a similar check here;
I like to see those when I'm `using Polymake`, but maybe I'm alone? @saschatimme, @benlorenz?

Reply:
I don't really care whether this is shown.

Reply:
ok, let's get rid of this logging

Reply:
@fieker instead of this, feel free to remove https://github.com/oscar-system/Polymake.jl/blob/fd04c00ecc27a282666acb07aeed32227d0b9914/src/generate_applications.jl#L14